### DISCUSSION TITLE
'message_hub' keyError when trying to finetune pretrained MMPose models.

### DISCUSSION THREAD

**Discussion by mmayank74567** [open-mmlab/mmpose#3076] [Q&A]
When resuming a model to finetune, MMengine expects to have a 'message_hub' key in the checkpoint. 
However, it seems like MMPose does not save the 'message_hub' when saving the torch model ([reference](https://github.com/open-mmlab/mmpose/pull/2036)) This causes the following issue:

```
  File "/mmengine/mmengine/runner/runner.py", line 2075, in resume
    self.message_hub.load_state_dict(checkpoint['message_hub'])
KeyError: 'message_hub'
```

**Comment by vieenrose** [comment]
Hi [mmayank74567](https://github.com/mmayank74567), 

We just got RTMO-s pre-trained weights reproduced in keeping all keys inside PyTorch checkpoint. 
You may check [Luigi/Retrainable-RTMO-s](https://huggingface.co/Luigi/Retrainable-RTMO-s) to get it if you need.
